src/utils/dom.ts

```diff
@@ -196,6 +205,10 @@ function queryMultiple<T extends Element = Element>(
   let matches = Array.from(element.querySelectorAll<T>(slotSelector));
   matches = options && options.direct === false ? matches : matches.filter((el) => el.parentElement === element);
 
+  if (slotSelector === defaultSlotSelector) {
+    matches = matches.filter((match) => match?.assignedSlot);
```

Comment:
@jcfranco It seems like this would work but we would need to move the spec tests to e2e for assignedSlot to work I think.